controllers/suite_util_test.go

```diff
@@ -300,6 +312,31 @@ func waitDoneMessageOnComponent(componentKey types.NamespacedName) {
 	}, timeout, interval).Should(BeTrue())
 }
 
+func expectPacBuildStatus(componentKey types.NamespacedName, state string, errID int, errMessage string, mergeURL string) {
+	buildStatus := readBuildStatus(getComponent(componentKey))
+	Expect(buildStatus).ToNot(BeNil())
+	Expect(buildStatus.PaC).ToNot(BeNil())
+	Expect(buildStatus.PaC.State).To(Equal(state))
+	Expect(buildStatus.PaC.ErrId).To(Equal(errID))
+	Expect(buildStatus.PaC.ErrMessage).To(Equal(errMessage))
+	if mergeURL != "" {
+		Expect(buildStatus.PaC.MergeUrl).To(Equal(mergeURL))
+	}
+}
```

Comment:
`ConfigurationTime` is not included in this function.

Reply:
ok I'll make it dependent on state

if enabled configurationtime is there
otherwise it is empty